mappings/net/minecraft/client/util/math/Quaternion.mapping

```diff
@@ -1,4 +1,31 @@
 CLASS net/minecraft/class_1158 net/minecraft/client/util/math/Quaternion
 	FIELD field_5656 components [F
+	METHOD <init> (FFFF)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (FFFZ)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (Lnet/minecraft/class_1158;)V
+		ARG 1 other
+	METHOD <init> (Lnet/minecraft/class_1160;FZ)V
+		ARG 1 direction
+		ARG 2 magnitude
+		ARG 3 normalize
 	METHOD equals (Ljava/lang/Object;)Z
 		ARG 1 o
+	METHOD method_16002 fSin (F)F
+		ARG 0 value
+	METHOD method_16003 fCos (F)F
+		ARG 0 value
+	METHOD method_4921 x ()F
+	METHOD method_4922 y ()F
+	METHOD method_4923 z ()F
+	METHOD method_4924 w ()F
```

Comment:
given they are getters, can we name them `getX()` `getY()` `getZ()` `getW()` instead?

Reply:
We can. Should we change the method on the vector classes too?